dGame/dGameMessages/GameMessages.cpp

```diff
@@ -3099,10 +3099,41 @@ void GameMessages::SendPropertySelectQuery(LWOOBJID objectId, int32_t navOffset,
 
 	bitStream.Write<uint32_t>(entries.size());
 
-	for (auto& entry : entries) {
-		entry.Serialize(bitStream);
-	}
+	for (const auto& item : entries) {
+		bitStream.Write(item.CloneID);
+
+		const auto& owner = GeneralUtils::UTF8ToUTF16(item.PrimaryData.OwnerName);
+		bitStream.Write(uint32_t(owner.size()));
+		for (uint32_t i = 0; i < owner.size(); ++i) {
+			bitStream.Write(static_cast<uint16_t>(owner[i]));
```

Comment:
I don't believe we need the static casts since owner is a u16string which has a CharT of char16_t and can likely be moved to the T type of the Write method instead since no conversion needs to happen. What do you think?

Reply:
will agree, the cast is unnecessary, since the are all already u16strings